src/inquirer3/questions.py

```diff
@@ -167,28 +165,54 @@ class Editor(Text):
 class Confirm(Question):
     kind = "confirm"
 
-    def __init__(self, name, default=False, trim_header=True, **kwargs):
-        super().__init__(name, default=default, trim_header=trim_header, **kwargs)
+    def __init__(
+        self,
+        name: Hashable,
+        message: Any = "",
+        default: Any = False,
+        ignore: Union[bool, Callable] = False,
+        validate: Union[bool, Callable] = True,
+        trim_header: Union[bool, Callable] = True,
```

Comment:
You forgot to add `show_default` (You fucking retard kid)

Reply:
Just looked at it right now, this is on purpose, no need `show_default` parameter on confirm question as it already shows the default behaviour